R-package/R/viz.graph.R

```diff
@@ -143,7 +142,7 @@ graph.viz <- function(symbol, shape=NULL, direction="TD", type="graph", graph.wi
   }
   
   graph<- create_graph(nodes_df = nodes_df_new, edges_df = edge_df_new, directed = TRUE) %>% 
-    set_global_graph_attrs("layout", value = "dot", attr_type = "graph") %>% 
+    add_global_graph_attrs("layout", value = "dot", attr_type = "graph") %>% 
```

Comment:
This is not the ideal fix though IMO. MXNet should still be able to run under DiagrammeR = 0.9.0. You should be able to check the version and trigger the methods properly based on different versions of DiagrammeR.

Reply:
This will potentially lose users who cannot afford to upgrade their DiagrammeR at the moment due to large codebases.

Reply:
Wouldn't it rather make sense to pin the version to 0.9.0 - 0.9.2?

Reply:
Or what do you think, @jeremiedb ? 0.9 sounds like an unstable beta version, so maybe it would make sense to tell users to upgrade? But we would also have to consider SemVer, so this might be difficult.

Reply:
Yes, agree that works in this case but I guess I was just commenting in general that this should be something to consider.

Reply:
Agree that I may have been too conservative with the 1.0.0 requirement. I just made a slight modification and graphs can now be rendered as before even on 0.9.0.